changelogs/fragments/environment_cli.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - ansible CLI tools that execute tasks can now pass a new option "-E" multiple times to define variables (k=v or YAML/JSON) to populate the environment keyword
```

Section prefix and full sentence:

```suggestion
  - CLI - Tools that execute tasks can now pass one or more ``-E`` options to define key-value pairs (k=v or YAML/JSON mapping) to be added to the ``environment`` task keyword for all hosts.
```